mahta/HTML/HTML tags.htm

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
1. Try to avoid using spaces in file names.
2. Most of the times, you name the main html file `index.html`.
3. `htm` is less used and `html` is preferred.
4. For other issues, please put the files of each issue in a directory, e.g. `mahta/HTML/HTMLTags/index.html`. It's not a big concern for HTML and you can ignore it, though, as it only has one associated issue.